tests/enospc-30mins.yaml

```diff
@@ -2,9 +2,9 @@ test_duration: 40
 n_db_nodes: 3
 n_loaders: 4
 n_monitor_nodes: 1
-user_prefix: 'gce-enospc-1-7'
+user_prefix: 'gce-enospc-not-jenkins'
```

Comment:
change gce-  to cases-

Reply:
how about `case-enospc-noJenkins`?

Reply:
GCE doesn't allow capital letters in names.